app/src/features/admin/users/ActiveUsersList.tsx

```diff
@@ -264,6 +315,38 @@ const ActiveUsersList: React.FC<IActiveUsersListProps> = (props) => {
           />
         )}
       </Paper>
+
+      <EditDialog
+        dialogTitle={'Add System Users'}
+        open={openAddUserDialog}
+        dialogSaveButtonLabel={'Add'}
+        component={{
+          element: (
+            <AddSystemUsersForm
+              system_roles={
+                props.codes?.system_roles?.map((item) => {
+                  return { value: item.id, label: item.name };
+                }) || []
+              }
+            />
+          ),
+          initialValues: AddSystemUsersFormInitialValues,
+          validationSchema: AddSystemUsersFormYupSchema
+        }}
+        onCancel={() => setOpenAddUserDialog(false)}
+        onSave={(values) => {
+          handleAddSystemUsersSave(values);
+          setOpenAddUserDialog(false);
+          dialogContext.setSnackbar({
+            open: true,
+            snackbarMessage: (
+              <Typography variant="body2" component="div">
+                {values.participants.length} team {values.participants.length > 1 ? 'members' : 'member'} added.
```

Comment:
Maybe double check with Jeremy what the wording here should be. "Team Members" is applicable at the project level, but probably not at the system level.